src/nn/wasm/src/CMakeLists.txt

```diff
@@ -9,15 +9,19 @@ include_directories(
   external/tensorflow/tensorflow/contrib/makefile/downloads/absl/
   external/tensorflow/tensorflow/contrib/makefile/downloads/eigen/
   external/tensorflow/tensorflow/contrib/makefile/downloads/gemmlowp/
+  external/tensorflow/tensorflow/lite/experimental/
   external/tensorflow/tensorflow/lite/kernels/
   external/tensorflow/third_party/
   )
 
 set(SOURCES
     bind/src/binding.cpp
+    external/tensorflow/tensorflow/lite/kernels/cpu_backend_context.cc
+    external/tensorflow/tensorflow/lite/experimental/ruy/allocator.cc
+    external/tensorflow/tensorflow/lite/experimental/ruy/thread_pool.cc
+    external/tensorflow/tensorflow/lite/kernels/cpu_backend_gemm_eigen.cc
    )
 
 add_executable(nn_ops ${SOURCES})
 set_property(TARGET nn_ops PROPERTY CXX_STANDARD 11)
-set_target_properties(nn_ops PROPERTIES LINK_FLAGS "-s WASM=1 -s NO_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1 -s SINGLE_FILE=1 -s MODULARIZE=1 --memory-init-file 0 --bind")
-
+set_target_properties(nn_ops PROPERTIES LINK_FLAGS "-s WASM=1 -s NO_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1 -s SINGLE_FILE=1 -s MODULARIZE=1 -s ASSERTIONS=1 --memory-init-file 0 --bind")
```

Comment:
What's the change for this line?

Reply:
Add the build option '-s ASSERTIONS=1'.

Reply:
Why do you need to add this option?

Reply:
To get the error massage if any error occur during the run, I can delete it if not needed.

Reply:
Please just keep that for your dev build. For production build, please remove that.